### PR TITLE
Add Agenda Jekyll templates

### PR DESCRIPTION
**Pull request by nchan0154**
### Related Issue
#5 

### Notes
I think I covered all the possible markup options. Please let me know if you want me to format/input the data, or add any additional documentation or placeholders.

### CODE REVIEW

File: agenda.html [hunk 1]
```diff
@@ -1,5 +1,163 @@
 ---
 permalink: /agenda
+layout: default
+schedule:
+- title: "Day 1: Presentation Day 1"
+- agenda:
+  - time: "08:15–08:45AM"
```
Comment (AmeliaBR) [inline agenda.html]: Shouldn't each day be a single YAML object with properties `title` and `agenda`? Meaning, no `-` in front of agenda?

**Review by AmeliaBR — COMMENTED** [review]
See quick question about the YAML format for the templates.

Other than that, looks great, thanks for the quick turnaround!

**Comment by AmeliaBR** [comment]
FYI, I merged in your fixes to the Gemfile/theme separately (commit https://github.com/Maps4HTML/Maps4HTML-Workshop-2020/commit/0d13e54160d4ce69ccb56416ec63f9d3314503c3), because I was having the exact same problem trying to make a local build & the site CSS kept on getting replaced by a reset file!!!!

File: agenda.html (hunk 1 above)
Comment (AmeliaBR) [reply agenda.html]: Pushed the fix myself after giving it a run-through.

It didn't look wrong in the displayed page, but yeah, you were getting one item with a title and then an empty agenda list, followed by an empty heading and the agenda.

Comment (AmeliaBR) [reply agenda.html]: Errr… trying to push. If I can figure out git.

Comment (nchan0154) [reply agenda.html]: Yep, you are correct, I think I accidentally appended a hyphen in there when copy and pasting the days. Good catch!

**Comment by AmeliaBR** [comment]
OK, I figured out how to push my fixes & edits to your branch. (I can never remember the syntax for pushing to a branch of a different name on the non-default remote…)

I'll add in slightly more useful placeholder agenda items myself & then merge it all in one go.

**Comment by nchan0154** [comment]
Thanks @AmeliaBR! My best guess is that this is a github-pages gem related thing, as it installs a bunch of theme gems including [this one where the CSS file is from](https://github.com/pages-themes/primer). I think it yoinked the CSS from this theme as the file paths were the same. I did not catch it when scaffolding it out initially as I had worked with vanilla Jekyll and not the github pages gem.  Truly, a mystery worthy of an Agatha Christie book. 😂

**Pull request merged**